index.html

```diff
@@ -10,7 +10,47 @@
     <link rel="stylesheet" href="styles/index.css" />
 </head>
 <body>
-    <script src="./src/index.js"></script>
+    <header class="header__header"> 
+        <h1>Weather Report</h1> 
+        <span>For the lovely city of <span id="headerCityName" class="header__city-name">Seattle</span>
+        </span> 
+    </header>
+    <section class="temp__container"> 
+        <h2>Temperature</h2> 
+        <div class="temperature__content"> 
+            <div class="temperature__controls"> 
+                <span id="increaseTempControl">⬆️</span> 
+                <span id="tempValue" class="orange"></span> 
+                <span id="decreaseTempControl">⬇️</span> 
+            </div> 
+            <button id="tempButton">Get Realtime Temperature</button> 
+        </div> 
+    </section>
+
+    <section class="sky__container"> 
+        <h2>Sky</h2> 
+        <select id="skySelect"> 
+            <option>Sunny</option> 
+            <option>Cloudy</option> 
+            <option>Rainy</option> 
+            <option>Snowy</option> 
+        </select> 
```

Comment:
[The `option` HTML tag should also have a `value` field](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select) to make these options accessible to screen readers: 
```
<select id="sky-menu">
    <option value="sunny">Sunny</option>
    <option value="cloudy">Cloudy</option>
    <option value="rainy">Rainy</option>
    <option value="snowy">Snowy</option>
</select>
```